### PR TITLE
Remove re-exports

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
cf. https://github.com/easystats/easystats/issues/292

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
Needs to wait for _parameters_ release.

**Comment by IndrajeetPatil** [comment]
@etiennebacher This should be ready for a review now since _parameters_ update is now on CRAN.

File: DESCRIPTION [hunk 1]
```diff
@@ -64,6 +64,8 @@ Suggests:
     testthat (>= 3.1.6),
     tidyr,
     withr
+Remotes:
+    easystats/parameters
```
Comment (etiennebacher) [inline DESCRIPTION:68]: do we need this now that parameters is on CRAN?

**Review by etiennebacher — CHANGES_REQUESTED** [review]

**Review by etiennebacher — APPROVED** [review]

**Comment by etiennebacher** [comment]
Thanks @IndrajeetPatil

**Pull request merged**